src/main/java/com/samoilenko/onlinebookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -39,6 +39,14 @@ protected ResponseEntity<Object> handleEntityNotFoundException(
         return getResponseEntity(HttpStatus.NOT_FOUND, bodyOfResponse);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    protected ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex, WebRequest request
+    ) {
+        String bodyOfResponse = "Can't register user" + ex.getMessage();
```

Comment:
`body` is confusing because I thought that it is the whole json body, when in fact it is just a message
```suggestion
        String errorMessage = "Can't register user" + ex.getMessage();
```